### ISSUE TITLE
Details about training target and loss function

### ISSUE THREAD

**Issue by chrisway613** [ali-vilab/MimicBrush#21]
Appreciate this excellent work! But I'm confused about the training target and loss function. 
According to the paper, training target is to recover the masked area of source image with diffusion model, whose "responsibility" is estimating the ground truth noise. So the loss function is noise prediction combined with masked area prediction?  What the loss function exactly? Can you tell me more details about it, I would be grateful!

**Comment by XavierCHEN34** [comment]
The loss is for the noise prediction of the full target image.
